src/datasources/asset/types/types.ts

```diff
@@ -9,3 +10,9 @@ export enum AssetQueryType {
 }
 
 export type AssetQuery = ListAssetsQuery | CalibrationForecastQuery | AssetSummaryQuery;
+
+export interface AssetDataSourceOptions extends DataSourceJsonData {
+  calibrationForecastEnabled: boolean | null;
+  assetListEnabled: boolean | null;
+  assetSummaryEnabled: boolean | null;
+}
```

Comment:
It'd be good if we had the default values for these toggles declared as well. Maybe something like:

```ts
interface AssetFeatureToggles {
  calibrationForecast: boolean;
  assetList: boolean;
  assetSummary: boolean;
}

interface AssetDataSourceOptions extends DataSourceJsonData {
  featureToggles: AssetFeatureToggles;
}

const AssetFeatureTogglesDefaults: AssetFeatureToggles = {
  calibrationForecast: false,
  assetList: false,
  assetSummary: true
}
```

Reply:
I've added your suggestions